web-client/public/js/upload.js

```diff
@@ -44,20 +42,6 @@ export const upload = function (sliderObject, sliderGroupController, drawGraph,
     sliders.updateValues();
     sliders.configureSliderControllers();
 
-    var settingsController = function () {
-        this.color = true;
-
-        this.setupSettingsHandlers = function () {
-            $(COLOR_PREFERENCES_CLASS).on("click", function () {
-                $(COLOR_PREFERENCES_CLASS).toggleClass(ACTIVE_COLOR_OPTION);
-                $(COLOR_PREFERENCES_CLASS + ">span").toggleClass("glyphicon-ok invisible");
-            });
```

Comment:
OK, so I see what happened here—you just moved this code to `updateApp`. That is generally the right idea, but don’t do so blindly either. Always keep the M->V->C cycle in mind.